models/articles.model.js

```diff
@@ -92,10 +92,33 @@ function incVotesById(article_id, inc_votes) {
   });
 }
 
+function deleteComment(comment_id) {
+  return db
+    .query("SELECT * FROM comments WHERE comment_id = $1;", [comment_id])
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "No such comment_id exist yet",
```

Comment:
I'd rewrite this error message a bit for clarity - saying something a bit more like 'No comment with specified Id' to make it clear what exactly has gone wrong

Reply:
Amended.